File: 0997-find-the-town-judge/0997-find-the-town-judge.java
class Solution {
    public int findJudge(int n, int[][] trust) {
        int[] t = new int[n + 1];
        for(int[] x : trust) {
            t[x[1]]++;
            t[x[0]]--;
        }
        for(int i = 1; i<=n; i++) {
            if(t[i] == n - 1) return i;
        }
        return -1;
    }
}

File: 0997-find-the-town-judge/0997-find-the-town-judge.ts
function findJudge(n: number, trust: number[][]): number {
    const points: number[] = Array(n).fill(0);
    trust.map((person: number[]) => {
        points[person[0] - 1] -= 1;
        points[person[1] - 1] += 1;
    });
    for(let per = 0; per < n; per++) {
        if(points[per] === n - 1) return per + 1;
    }
    return -1;
};